boot/freeldr/freeldr/include/arch/pc/hardware.h

```diff
@@ -72,6 +72,11 @@ VOID
 DetectBiosDisks(PCONFIGURATION_COMPONENT_DATA SystemKey,
                 PCONFIGURATION_COMPONENT_DATA BusKey);
 
+#if defined(_M_IX86)
+VOID __cdecl HwIrqHandler(VOID);
+extern volatile ULONG HwIrqCount;
+#endif
```

Comment:
I wonder whether these two things need to be in the global hardware.h header, or could be placed in a localized one (within arch/i386/pc) or even just in the (single??) .c file that would use those.

Reply:
Moved into a single .c file